Cloudinary/Classes/Core/Features/ManagementApi/CLDManagementApi.swift

```diff
@@ -57,8 +59,8 @@ import Foundation
                             as well as performing actions on the request, such as cancelling, suspending or resuming it.
     */
     @discardableResult
-    open func rename(_ publicId: String, to: String, overwrite: Bool? = nil, invalidate: Bool? = nil, params: CLDRenameRequestParams? = nil, completionHandler: ((_ result: CLDRenameResult?, _ error: Error?) -> ())? = nil) -> CLDRenameRequest {
-        let renameParams = CLDRenameRequestParams(fromPublicId: publicId, toPublicId: to, overwrite: overwrite, invalidate: invalidate)
+    open func rename(_ publicId: String, to: String, overwrite: Bool? = nil, invalidate: Bool? = nil, params: CLDRenameRequestParams? = nil, context: Bool? = nil, metadata: Bool? = nil, completionHandler: ((_ result: CLDRenameResult?, _ error: Error?) -> ())? = nil) -> CLDRenameRequest {
```

Comment:
@const-cloudinary I think we should put the new parameters only in the `CLRenameRequestParams` and not part of the `rename` function

Reply:
@AlexPinhasovLili  please put the `context` and `metadata` params only in the `CLDRenameRequestParams` and not part of the `rename` function